apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimTransformerV2.java

```diff
@@ -218,16 +218,22 @@ private static Reference getPatient(PreAdjFissClaim claimGroup) {
 
     for (int i = 0; i < procCodes.size(); ++i) {
       PreAdjFissProcCode procCode = procCodes.get(i);
-      Claim.ProcedureComponent component = new Claim.ProcedureComponent();
-
-      component.setSequence((i + 1));
-      component.setDate(
-          Date.from(procCode.getProcDate().atStartOfDay(ZoneId.systemDefault()).toInstant()));
-      component.setProcedure(
-          new CodeableConcept()
-              .setCoding(
-                  Collections.singletonList(
-                      new Coding("http://hl7.org/fhir/sid/icd-10", procCode.getProcCode(), null))));
+      Claim.ProcedureComponent component =
```

Comment:
@KyRobbins can we add a unit test for the `getProcedure` method to show it won't raise a null pointer when the proc date is null